Thanks for this @fiirhok. It looks like there's a failing test:
```
--- FAIL: TestBasic (0.00s)
    mustache_test.go:222: "{{ a }}{{=<% %>=}}<%b %><%={{ }}=%>{{ c }}" expected "abc" but got error "line 1: empty tag"
    mustache_test.go:222: "{{ a }}{{= <% %> =}}<%b %><%= {{ }}=%>{{c}}" expected "abc" but got error "line 1: empty tag"
panic: runtime error: slice bounds out of range [1:0] [recovered]
	panic: runtime error: slice bounds out of range [1:0]
```

--- a/mustache.go
+++ b/mustache.go
@@ -437,10 +437,6 @@ func (tmpl *Template) parse() error {
 			}
 			tmpl.elems = append(tmpl.elems, partial)
 		case '=':
-			if tag[len(tag)-1] != '=' {
-				return newError(tmpl.curline, ErrInvalidMetaTag)
-			}
-			tag = strings.TrimSpace(tag[1 : len(tag)-1])
 			tmpl.parseCustomDelimiter(tag)
 		case '{':
 			//use a raw tag

